internal/autothrottle/replication/throttles_update.go

```diff
@@ -190,17 +190,13 @@ func (tm *ThrottleManager) UpdateOverrideThrottles() error {
 	var toRemove = make(map[int]struct{})
 
 	for _, override := range tm.brokerOverrides {
-		// ReassignmentParticipant have already had their override rate used as part
-		// of an ongoing reassignment.
-		if !override.ReassignmentParticipant {
-			rate := float64(override.Config.Rate)
-			// Rate == 0 means the rate was removed via the API.
-			if rate == 0 {
-				toRemove[override.ID] = struct{}{}
-			} else {
-				toAssign[override.ID] = struct{}{}
-				capacities.storeLeaderAndFollerCapacity(override.ID, rate)
-			}
+		rate := float64(override.Config.Rate)
+		// Rate == 0 means the rate was removed via the API.
+		if rate == 0 {
+			toRemove[override.ID] = struct{}{}
+		} else {
+			toAssign[override.ID] = struct{}{}
+			capacities.storeLeaderAndFollerCapacity(override.ID, rate)
```

Comment:
This looks like it updates `leader.replication.throttled.rate` and `follower.replication.throttled.rate` only. What happens to the value of `replication.throttled.replicas`?

The desired state is that, for each topic, `replication.throttled.replicas` is set to be the union of whatever it should be for the reassignment, plus every replica on the broker with the override. 

I'm guessing this is what should happen on L222 below, but I would expect that if the replaced broker was part of a a reassignment then `tm.skipOverrideTopicUpdates` is `true` (so we skip), because of [this code](https://github.com/DataDog/kafka-kit/blob/b298cd9710e42e226f92d2edfd5020e54c0b7c87/cmd/autothrottle/main.go#L353).